src/expr.rs

```diff
@@ -549,6 +550,21 @@ pub mod parsing {
         ))
     ));
 
+    named!(expr_loop -> Expr, do_parse!(
+        id: option!(
+            terminated!(
+                preceded!(
+                    punct!("'"),
+                    ident),
+                punct!(":"))) >>
```

Comment:
Never mind this comment, I misread the parentheses. The one on the next line applies though.